custom/auth/ante/burntax.go

```diff
@@ -47,11 +51,27 @@ func (btfd BurnTaxFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 
 	if !simulate {
 		// Compute taxes again.
-		taxes := FilterMsgAndComputeTax(ctx, btfd.treasuryKeeper, msgs...)
+		taxes := FilterMsgAndComputeTax(ctx, btfd.TreasuryKeeper, msgs...)
 
 		// Record tax proceeds
 		if !taxes.IsZero() {
-			err = btfd.bankKeeper.SendCoinsFromModuleToModule(ctx, types.FeeCollectorName, treasury.BurnModuleName, taxes)
+			if currHeight >= TaxPowerSplitHeight {
+				feePool := btfd.DistributionKeeper.GetFeePool(ctx)
+
+				for _, taxCoin := range taxes {
+					splitCoin := sdk.NewCoin(taxCoin.Denom, taxCoin.Amount.Quo(sdk.NewInt(2)))
```

Comment:
`Quo(sdk.NewInt(2)))` should be a chain param. We can put it into the treasury module.